miso-web/src/main/webapp/scripts/list.js

```diff
@@ -97,6 +97,7 @@ ListUtils = (function() {
           'bAutoWidth' : false,
           'iDisplayLength' : 25,
           'iDisplayStart' : 0,
+          'sScrollX' : '100%',
```

Comment:
Is this supposed to add a horizontal scrollbar instead of overflowing list contents off the right side of the screen? It doesn't seem to be working if so